polaris-core/src/main/java/org/apache/polaris/core/connection/ConnectionType.java

```diff
@@ -35,6 +35,9 @@ public enum ConnectionType {
   HADOOP(2),
   ;
 
+  public static final String ICEBERG_REST_FACTORY_IDENTIFIER = "ICEBERG_REST";
+  public static final String HADOOP_FACTORY_IDENTIFIER = "HADOOP";
```

Comment:
THose are superfluous, as those are the enum names, and can therefore be removed.